src/puppetlabs/puppetdb/cli/time_shift_export.clj

```diff
@@ -163,9 +200,10 @@
   (let [input-path (:input args)
         newest-timestamp (newest-timestamp-from-archive input-path)
         shift-period (.toPeriod (interval newest-timestamp (:shift-to-time args)))
-        output-path (:output args)]
+        output-path (:output args)
```

Comment:
here we should change the parameters inside args and pass them, instead of creating a variable for each parameter